plenum/test/restart/test_backup_primary_does_not_restore_pp_seq_no_if_view_is_not_same.py

```diff
@@ -0,0 +1,94 @@
+from plenum.test import waits
+from plenum.test.helper import sdk_send_batches_of_random, assertExp, \
+    sdk_send_random_and_check
+from plenum.test.pool_transactions.helper import \
+    disconnect_node_and_ensure_disconnected
+from plenum.test.test_node import ensureElectionsDone, getPrimaryReplica, \
+    getRequiredInstances
+from plenum.test.view_change.helper import ensure_view_change, \
+    start_stopped_node
+from stp_core.loop.eventually import eventually
+
+nodeCount = 4
+
+backup_inst_id = 1
+
+
+def test_backup_primary_does_not_restore_pp_seq_no_if_view_is_not_same(
```

Comment:
The test looks too complex to just check that backup primary does not restore pp_seq_no if viewNo is not the same.
Why do we need to restart all nodes here?